From edf23ae8816486f1978f712a0a280320ff493769 Mon Sep 17 00:00:00 2001
From: Krzysztof Kolasa <kkolasa@winsoft.pl>
Date: Tue, 1 Oct 2013 20:30:28 +0200
Subject: [PATCH] Add string GPL0 for kernel 3.12.0-rc3

---
 firegl_public.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/firegl_public.c b/firegl_public.c
index a47cf7d..d2bc214 100644
--- a/firegl_public.c
+++ b/firegl_public.c
@@ -259,7 +259,11 @@
 #endif
 
 #ifdef MODULE_LICENSE
+#if LINUX_VERSION_CODE < KERNEL_VERSION(3,12,0)
 MODULE_LICENSE("Proprietary. (C) 2002 - ATI Technologies, Starnberg, GERMANY");
+#else
+MODULE_LICENSE("GPL\0Proprietary. (C) 2002 - ATI Technologies, Starnberg, GERMANY");
+#endif
 #endif
 #ifdef MODULE_DEVICE_TABLE
 MODULE_DEVICE_TABLE(pci, fglrx_pci_table);
-- 
1.8.4

