diff -Npur fglrx-installer_15.200.orig/lib/modules/fglrx/build_mod/kcl_str.c fglrx-installer_15.200/lib/modules/fglrx/build_mod/kcl_str.c
--- fglrx-installer_15.200.orig/lib/modules/fglrx/build_mod/kcl_str.c	2015-03-06 17:44:48.000000000 +0100
+++ fglrx-installer_15.200/lib/modules/fglrx/build_mod/kcl_str.c	2015-04-13 12:57:26.126123367 +0200
@@ -42,6 +42,10 @@
 #include "kcl_type.h"
 #include "kcl_str.h"
 
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(4,0,0))
+#define strnicmp strncasecmp
+#endif
+
 /** \brief Fill memory with a constant byte
  *  \param s Pointer to memory
  *  \param c Initializing value
