From f22799e2023df7958a07ae240558d013269726dd Mon Sep 17 00:00:00 2001
From: Krzysztof Kolasa <kkolasa@winsoft.pl>
Date: Mon, 27 Apr 2015 19:46:42 +0200
Subject: [PATCH] Changes for kernel 4.1.0, remove IRQF_DISABLED

---
 firegl_public.c | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/firegl_public.c b/firegl_public.c
index 44706d9..e38cebf 100644
--- a/firegl_public.c
+++ b/firegl_public.c
@@ -3505,8 +3505,12 @@ int ATI_API_CALL KCL_InstallInterruptHan
         ((useMSI) ? (SA_INTERRUPT) : (SA_SHIRQ)),
 #else
         //when MSI enabled. keep irq disabled when calling the action handler,
-        //exclude this IRQ from irq balancing (only on one CPU) 
-        ((useMSI) ? (IRQF_DISABLED) : (IRQF_SHARED)),    
+        //exclude this IRQ from irq balancing (only on one CPU)    
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4,1,0)
+        ((useMSI) ? (IRQF_DISABLED) : (IRQF_SHARED)),
+#else
+        ((useMSI) ? (IRQF_NOBALANCING) : (IRQF_SHARED)),
+#endif
 #endif
         dev_name,
         context);
